internal/psximager/psxinject.go

```diff
@@ -25,7 +26,10 @@ func newInjectCmd(psximagerPath, cueFile, orgFile, injectFile string) *exec.Cmd
 func Execute(psximagerPath, cueFile, orgFile, injectFile string) error {
 	cmd := newInjectCmd(psximagerPath, cueFile, orgFile, injectFile)
 	if err := cmd.Run(); err != nil {
-		return fmt.Errorf("error executing %s: %w", PsxInject, err)
+		if stderr, ok := cmd.Stderr.(*bytes.Buffer); ok {
```

Comment:
Needs to be covered.

Reply:
[Done.](https://github.com/ChristoWolf/vagrant-story-randomizer/commit/c2565077efef918dff8f4b871b64b3b7cfbd82ea)